deepfence_frontend/apps/dashboard/src/features/postures/pages/PostureCloudScanResults.tsx

```diff
@@ -1125,12 +1140,12 @@ const Filters = () => {
           {benchmarks
             .filter((item) => {
               if (!benchmarkQuery.length) return true;
-              return item.toLowerCase().includes(benchmarkQuery.toLowerCase());
+              return item.includes(benchmarkQuery);
```

Comment:
Why this particular line is changed? This is for filtering the dropdown items with users search query.

Reply:
revert in cloud and host result